src/components/layout/ApplicantsLayout.tsx

```diff
@@ -1,27 +1,128 @@
-import { Stack } from '@chakra-ui/react';
-import { FC } from 'react';
-import { ApplicantsDescription, ReadyToApply } from '../UI/common';
+import { Flex, Stack, Tab, TabList, Tabs } from '@chakra-ui/react';
+import { useRouter } from 'next/router';
+import { FC, useState } from 'react';
 
-interface Props {
-  linkToApply: string;
-}
+import { ApplicantsDescription, ImportantText, ReadyToApply } from '../UI';
+
+import {
+  APPLICANTS_PAGES_BASEPATH,
+  APPLICANTS_TABS,
+  APPLICANTS_URL,
+  OFFICE_HOURS_URL,
+  PROJECT_GRANTS_URL,
+  SMALL_GRANTS_URL,
+  TABS_MAP
+} from '../../constants';
+
+export const ApplicantsLayout: FC = ({ children }) => {
+  const router = useRouter();
+  const [tabIndex, setTabIndex] = useState(TABS_MAP[router.pathname]);
+
+  const handleChange = (index: number) => {
+    setTabIndex(index);
+
+    switch (index) {
+      case 0:
+        router.push(
```

Comment:
As I mentioned on that call, I think it'd be ideal for these tabs to actually be links given that we replicate the same functionality (i.e. clicking a tab routes the user to that route).

Is this something the `Tabs` Chakra component can support? Or would this require significant changes?

In any case, I see this as a "nice to have", not a blocker.

Reply:
Yes, it's supported, rendering the tab as an `<a>` and adding cursor pointer.

<img width="380" alt="Screen Shot 2022-01-10 at 16 06 39" src="https://user-images.githubusercontent.com/948922/148824364-e5f854e5-e90b-4d5c-acfc-a3f9edcce0b8.png">

<img width="1069" alt="Screen Shot 2022-01-10 at 16 03 04" src="https://user-images.githubusercontent.com/948922/148824084-f183227f-dbe7-42b2-8db0-625352f5ddd0.png">